src/bootstrap/compile.rs

```diff
@@ -162,7 +162,11 @@ pub fn std_cargo(build: &Build,
         // missing
         // We also only build the runtimes when --enable-sanitizers (or its
         // config.toml equivalent) is used
-        cargo.env("LLVM_CONFIG", build.llvm_config(target));
+        let llvm_config = build.ensure(native::Llvm {
+            target: build.config.build,
```

Comment:
Shouldn't this be the target we're passing to std_cargo? Or is this some other llvm-config? Either way, it'd be good to comment on the build.config.build with why we're passing it, and not `target`.

It also looks like it's maybe worth making this `build.ensure(...)` call the body of the old `llvm_config` function and then just calling it so we avoid duplicating it. But that's probably a minor detail, up to you.

Reply:
Ah good point! I was hoping to keep the `ensure` here though to canonicalize the definition of where `llvm-config` comes from in one place (previously it was calculated in two places)